test/test_reductions.py

```diff
@@ -1227,6 +1227,17 @@ def _amax_wrapper(x, dim=None, keepdims=False):
 
         self._test_minmax_helper(_amin_wrapper, np.amin, device, dtype)
         self._test_minmax_helper(_amax_wrapper, np.amax, device, dtype)
+    
+    @onlyNativeDeviceTypes
+    @dtypes(*complex_types_and(torch.bool))
+    def test_invalid_0dim_aminmax(self, device, dtype):
+
+        with self.assertRaisesRegex(RuntimeError, 'not implemented'):
+            if dtype is bool:
+                torch.aminmax(torch.tensor(1, dtype=dtype, device=device), dim=0) 
+            else:
+                torch.aminmax(torch.tensor(1, dtype=dtype, device=device), dim=0)
```

Comment:
```suggestion
            torch.aminmax(torch.tensor(1, dtype=dtype, device=device), dim=0) 
```
the if and else statements look identical